release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1258,6 +1258,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * Previously, monitoring alerts with severity levels such as `critical` and `warning` were treated as `info` level alerts. As a result, the *Monitoring Alert* icon was not displayed on the workload in the *Topology* view for these alerts. This issue is now fixed; alerts like `critical` are treated as `warning` level alerts and a *Monitoring Alert* icon is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
 
+* Previously, in the *YAML view* of the Helm installation form only the YAML schema or code was shown. There was no option to provide and view comments with parameters. Now you can view parameters in a defined order with comments. A *Schema* viewer is added to the page, next to the *YAML* editor to check all the schema objects present in the Helm Chart. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886861[*BZ#1886861*])
```

Comment:
YAML schema was not shown earlier. Only YAML code was shown. This bug fix adds the schema viewer in yaml editor. Also, the comments are not shown in YAML code. Only the schema and its descriptions.

Reply:
Thanks for the clarification @rohitkrai03. Will update the content based on this comment.